tests/test_lib.py

```diff
@@ -257,6 +257,93 @@ def test_create_workspace(self, requests_mock):
         )
         assert api_response == self.api_client.create_workspace(name, ORGANIZATION_URL, metadata)
 
+    @pytest.mark.usefixtures("mock_login_request")
+    def test_get_annotations(self, requests_mock):
+        api_response_pages = {
+            f"{ANNOTATIONS_URL}?queue=0&status=exported&sideload=documents": {
```

Comment:
Given that `queue=0&status=exported&&sideload=documents` is repeated for each call, consider making it a variable for better readability